.env.exmaple

```diff
@@ -0,0 +1,7 @@
+SD_BACKEND_URL="/v2"
```

Comment:
As an example, it should be more detailed. I'm suggesting to put here something like 

`SD_BACKEND_URL="http://localhost:8080/v2"`